R/ci.R

```diff
@@ -1,82 +1,104 @@
 #' Continuous integration setup and badges
 #'
 #' Sets up continuous integration (CI) services for an R package that is
-#' developed on GitHub. CI services can run `R CMD check` automatically on
-#' various platforms, triggered by each push or pull request. These functions
-#' * Add service-specific configuration files and add them to `.Rbuildignore`.
-#' * Activate a service or give the user a detailed prompt.
-#' * Provide the markdown to insert a badge into README.
+#' developed on GitHub with CI-agnostic support by \pkg{tic}.
+#' CI services can run `R CMD check` automatically on
+#' various platforms, triggered by each push or pull request. This function
+#' * Adds service-specific configuration files and adds them to `.Rbuildignore`.
+#' * Activates a service or gives the user a detailed prompt.\cr
 #'
-#' @name ci
-#' @aliases NULL
-NULL
-
-#' @section `use_travis()`:
-#' Adds a basic `.travis.yml` to the top-level directory of a package. This is a
-#' configuration file for the [Travis CI](https://travis-ci.org/) continuous
-#' integration service.
-#' @param browse Open a browser window to enable automatic builds for the
-#'   package.
+#' @details
+#' By default the CI-services "Travis" (Linux) and "Appveyor"
+#' (Windows) will be set up. Basic `.travis.yml` and `appveyor.yml` files are
+#' added to the top-level directory of a package.
+#'
+#' This function is aimed at supporting the most common use cases.
+#' Users who require more control are advised to manually call the individual
+#' functions.
+#'
+#' @name use_ci
+#' @importFrom travis travis_set_pat
+#' @param path `[string]`\cr
+#'   The path to the repo to prepare.
+#' @param quiet `[flag]`\cr
+#'   Less verbose output? Default: `FALSE`.
+#' @param services `[character]`\cr
+#'   CI services to add.
 #' @export
 #' @rdname ci
-use_travis <- function(browse = interactive()) {
```

Comment:
Do we want to keep `use_travis()` and `use_appveyor()` and forward to `use_ci()` (with a message/warning)?

Reply:
And deprecate? Or remove the `services` arg again from `use_ci()`?

Reply:
I think _tic_ needs to gain more traction before we can deprecate. At some point `r.rb` from Travis might look for a `tic.R` file and call into _tic_, that would be a good time to start deprecating other approaches.

Reply:
Ok, I re-added both and put them into one help file with deprecation warnings.